setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     scipy >= 1.6.0
     nibabel >= 3.0
     h5py
+    pathlib
```

Comment:
I know this has been merged in for a while, but why include `pathlib` in `install_requires` when https://github.com/nipy/nitransforms/blob/f69faaf44a9cab39141df6f546394e862ed408d2/setup.cfg#L24 and pathlib was incorporated into the standard Python library in v3.4? This requirement overrides the current-Python-version's `pathlib` with the version on PyPI for Python 3.3, which breaks `pip` (at least in C-PAC's current release Dockerfile, which [includes](https://github.com/FCP-INDI/C-PAC/blob/96db8b0b65ab1d5f55fb3b895855af34d72c17e4/requirements.txt#L36)

```pip-requirements
sdcflows==2.4.0
```

which [requires](https://github.com/nipreps/sdcflows/blob/188bc87a79e89002fbccd13a4c701e3bd2224822/requirements.txt#L7)
```pip-requirements
nitransforms>=21.0.0
```
)